packages/web-runtime/tests/unit/helpers/theme.spec.ts

```diff
@@ -21,28 +17,28 @@ describe('theme loading and error reporting', () => {
   })
 
   it('should load the default theme if location is not found', async () => {
-    fetch.mockResponse(new Error(), { status: 404 })
+    ;(fetch as FetchMock).mockResponse(new Error() as any, { status: 404 })
```

Comment:
I found no method with a signature that allows me to pass an error and a status ... hence I need to cast error :-\